src/test/groovy/unit/org/pih/warehouse/utils/DateUtilSpec.groovy

```diff
@@ -51,27 +51,26 @@ class DateUtilSpec extends Specification {
         "01/01/2000 00:00 +0500"    || "1999-12-31T19:00:00Z" | "Timezone conforming to XX format w/ positive our format"
         "01/01/2000 00:00 +05:00"   || "1999-12-31T19:00:00Z" | "Timezone conforming to XXX format w/ positive our format"
 
-        // This case fails because when no timezone is provided to Date, it uses the system local timezone instead of
-        // UTC. This can cause unexpected behaviour! Adding "TimeZone.setDefault(TimeZone.getTimeZone(ZoneOffset.UTC))"
-        // to Bootstrap.groovy will change the default to always be UTC. This fixes the issue by making the behaviour
-        // consistent (and would allow us to re-enable this test).
-        //"01/01/2000"                || "2000-01-01T00:00:00Z" | "No time or timezone our format"
+        // When not given a timezone, we default to UTC.
+        "01/01/2000 00:00"          || "2000-01-01T00:00:00Z" | "No timezone our format"
+        "2000-01-01 00:00"          || "2000-01-01T00:00:00Z" | "No timezone ISO format"
 
-        // It's noteworthy that the "yyyy" format for java.util.Date supports a two year format but it is year 0 based,
-        // not based on the year of this century (ie "25" is year 25, not 2025)!
-        "01/01/00 00:00 Z"          || "0001-01-01T00:00:00Z" | "No time or timezone our format two digit year"
+        // When not given a time or timezone, we default to midnight UTC.
+        "01/01/2000"                || "2000-01-01T00:00:00Z" | "No time or timezone our format"
+        "2000-01-01"                || "2000-01-01T00:00:00Z" | "No time or timezone ISO format"
```

Comment:
we can parse to a Date without providing a timezone and it automatically uses UTC now. Yay!